apis/speech/v1beta1/phraseset_types.go

```diff
@@ -40,7 +40,7 @@ type SpeechPhraseSetSpec struct {
 	//  binary search approach to finding the optimal value for your use case as
 	//  well as adding phrases both with and without boost to your requests.
 	// +kcc:proto:field=google.cloud.speech.v2.PhraseSet.boost
-	Boost *string `json:"boost,omitempty"`
+	/* NOTYET Boost *string `json:"boost,omitempty"` */
```

Comment:
If we are removing fields, we should do it in alpha first?

Reply:
Removing fields is a backward-incompatible change. To avoid the need for a conversion webhook, we typically remove fields while the CRD is still in alpha. This ensures the schema remains unchanged when promoting to beta.

Reply:
removed the fields from Alpha (KCC users does not roll back from Beta to Alpha  ¯\_(ツ)_/¯ ).